src/sign.ts

```diff
@@ -177,4 +178,16 @@ export class SignJWT extends ProduceJWT {
       protectedHeader,
     };
   }
+
+   /**
+   * Decodes a JWT without signature
+   *
+   * @param jwtWithoutSignature JWT to sign that needs to be decoded.
+   */
+  static encode(jwt: {
+    payload: JWTPayload;
+  header: JWSHeaderParameters;
+  }): string {
+    return [jwt.header, jwt.payload].map(e =>JSON.stringify(e)).map(encodeBase64).join(".")
```

Comment:
is decode or encode? This looks to me like the code of an encode.

Reply:
It's actually a typo, shouldn't be in this PR. I remove it